Managers/LuaMan.cpp

```diff
@@ -403,6 +405,38 @@ namespace RTE {
 		return isDefined;
 	}
 
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+	const std::vector<std::string> & LuaMan::DirectoryList(const std::string &filePath) {
+		std::string fullPath = System::GetWorkingDirectory() + filePath;
```

Comment:
What happens in these methods when the path is invalid or doesn't exist?

Reply:
It prints 
```
ERROR: [string "a = LuaMan:DirectoryList("./x")"]:1: directory_iterator::directory_iterator: The system cannot find the path specified.: "I:/Programming/Cortex-Command-Community-Project-Data/./x"
```
I was thinking this was fine since a mod author should quickly be able to spot it, but what is your preferred behavior?

Reply:
Good enough.